### PR TITLE
Fix PortHandler initialization

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: #3050

### CODE REVIEW

**Review by McGiverGim — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by howels** [comment]
Latest push has the same behaviour as before - TCP port shows up for a fraction of a second then vanishes.

File: src/js/port_handler.js
Comment (haslinghuis) [inline src/js/port_handler.js:71]:
```suggestion
            ...(MdnsDiscovery?.mdnsBrowser.services?.filter(s => s.txt.vendor === 'elrs' && s.txt.type === 'rx')
```

**Comment by haslinghuis** [comment]
Closing as development is continued on https://github.com/betaflight/betaflight-configurator/pull/3095

**Pull request closed without merging**
